adafruit_character_lcd/character_lcd.py

```diff
@@ -620,9 +620,12 @@ class Character_LCD_RGB(Character_LCD):
     :param ~digitalio.DigitalInOut db7: The data line 7
     :param columns: The columns on the charLCD
     :param lines: The lines on the charLCD
-    :param ~pwmio.PWMOut, ~digitalio.DigitalInOut red: Red RGB Anode
-    :param ~pwmio.PWMOut, ~digitalio.DigitalInOut green: Green RGB Anode
-    :param ~pwmio.PWMOut, ~digitalio.DigitalInOut blue: Blue RGB Anode
+    :param red: Red RGB Anode
+    :type red: ~pwmio.PWMOut or ~digitalio.DigitalInOut
+    :param green: Green RGB Anode
+    :type green: ~pwmio.PWMOut or ~digitalio.DigitalInOut
+    :param blue: Blue RGB Anode
+    :type blue: ~pwmio.PWMOut or ~digitalio.DigitalInOut
```

Comment:
First of all, before I review all this and I forget thanks for this :)
This should be as previously, this is the preferred way of documenting the libraries, has been used all across the CircuitPython Libraries
For reference https://circuitpython.readthedocs.io/en/latest/docs/design_guide.html

Reply:
This was changed because the old way doesn't work:
<img width="345" alt="Old docs" src="https://user-images.githubusercontent.com/70200140/118341796-6b5c8480-b518-11eb-8e45-84702a2e0cfb.png">

New way:
<img width="350" alt="New docs" src="https://user-images.githubusercontent.com/70200140/118341799-6d264800-b518-11eb-869e-60d0dcd44e9c.png">

I would usually try to do something like `:param Union[typeA, typeB] parameterName: description` but I couldn't get it to work, possibly because of the `~`s.

Reply:
Fair enough, lets merge, I remember that happens to me also, the unions are not longer working, I guess with the new Sphinx version. I would try to troubleshoot, if we could not figure out we would need to put a note in the design guide